src/main/java/nextstep/courses/infrastructure/JdbcImageRepository.java

```diff
@@ -0,0 +1,42 @@
+package nextstep.courses.infrastructure;
+
+import nextstep.courses.domain.*;
+import org.springframework.jdbc.core.JdbcOperations;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDateTime;
+
+@Repository("imageRepository")
+public class JdbcImageRepository implements ImageRepository {
+    private JdbcOperations jdbcTemplate;
+
+    public JdbcImageRepository(JdbcOperations jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    @Override
+    public int save(Image image, Long sessionId) {
+        String sql = "insert into image (session_id, size, image_type, width, height, created_at) values(?, ?, ?, ?, ?, ?)";
+        return jdbcTemplate.update(sql, sessionId, image.getImageSize().getSize(), image.getImageType().name(), image.getImagePixel().getWidth(), image.getImagePixel().getHeight(), LocalDateTime.now());
+    }
+
+    @Override
+    public Image findById(long id) {
+        String sql = "select id, size, image_type, width, height from image where id = ?";
+        RowMapper<Image> rowMapper = (rs, rowNum) -> {
+            long imageId = rs.getLong(1);
+            long size = rs.getLong(2);
```

Comment:
```suggestion
            long imageId = rs.getLong("id");
            long size = rs.getLong("size");
```
이름으로도 매핑시켜보면 좋을 것 같아요
바인딩 수가 많아진다면 숫자로 매핑하기에는 어려움이 있을 것 같아요

`NamedParameterJdbcTemplate`에 대해 검색하고 공부하면 좋을 것 같네요!